functions/functions.inc

```diff
@@ -101,5 +101,8 @@ function search_definedFields($key)
 // Start the session so we are ready to go no matter what we do!
 session_start();
 
+// Include the Master Database
+require_once($FUNCTIONDIR."/database.inc");
```

Comment:
If we have backends somewhere else then we would want to define a $BACKENDDIR

I feel like we can start with that here and then move neon.inc there. Of vis versa we could move neon.inc first then add the sql stuff. I am indifferent.

Reply:
yes